### DISCUSSION TITLE
Confirmed not working (yet) - Akai S5000 Sampler

### DISCUSSION THREAD

**Discussion by telmnstr** [erichelgeson/BlueSCSI-v1#157] [Q&A]
Tried BlueSCSI on a S5000 sampler. 

Termination off, device is powering itself from the SCSI bus.  First tried a card with a 16GB file (HD40_512.hda) where the file had no filesystem in it. Sampler crashed on boot. Tried again with 2GB file, one with no filesystem and one with a partition table and a Fat32 partition. Figured I would format the partition from the machine to test write.

On the Sampler when listing disks under disk utilities menu or whatever it never shows any drives from the BlueSCSI, but can see other devices on the bus. Tried with only the bluescsi and termination on as well. So it is not acknowledging it's there yet.

Will keep poking around at it as time permits.

**Comment by telmnstr** [comment]
Progress has been made!

Removed termination jumper 1, left 2nd one on and S5000 sees the BlueSCSI. Saw this in another thread.

Formatting disk and full drive copy works, but after it finishes the S5000 didn't exit back to menu. It seemed locked up, probably waiting for a reply from the drive. It kind of sounded like it issued something that hit the spinning drive afterwards, could it tell disks to all reset? I heard the heads park or something on the spinning disk. Power cycling it, the disk seemed usable and properly formatted.

Recorded a 20 second empty sample and saved it to a SD card, this worked fine and no issues with it jumping back to menu afterwards.

Loading data, the original 2GB IBM brand SCSI disk data set that takes around 55 seconds from spinning disk takes around 1:17 from the BlueSCSI. So a bit less throughput but the silence and data portability (especially for those without the USB interface on the machine) is worth it. Data set I think was 40MB.

Next I will figure out how I plan to mount it, and keep access to the card. If I 3d print a mount I will post it on thingiverse.

I plan to keep an eye on the NeXT thread as I have one that could benefit from a BlueSCSI. I will try to test BlueSCSI on an Atari TT030 soon.

**Comment by baritonomarchetto** [comment]
The BlueSCSI v2 works perfectly for me on akai S5000. Used a 32 Gb image created with disk jokey. As far as I know, this is the maximum capacity for this sampler. I cannot exclude 4x 8 Gb images each could work (not tester yet), but for sure only one 32Gb can be handled by the sampler.

I think it's time to confirm BlueSCSI V2 actually WORKS on this sampler ;)

**Reply by dlmorley** [reply]
Which settings did you use in Disk Jockey? In the Akai section it only allows 10GB or did you just use the "other" section?
Thanks!

**Comment by SynthfulDwarphus** [comment]
I've seen threads in past re huge SSDs (100s of GB) on S5/6k, and excerpt below from current Reverb listing for S5000:

"This unit ships with a 512GB SATA SSD, and recognizes and addresses 482GB (as seen in photos). Drive is removable, system compatible with most best of brand SATA drives, with ease"

**Reply by baritonomarchetto** [reply]
S5000/6000 are not SATA compatibile, so I think there must be some issue in the listing. 
About HD>32Gb, it should be tested if they are actually usable to stiore more than 32Gb of data. It's a FAT32 limit

**Comment by SynthfulDwarphus** [comment]
It's old news, apparently using adapter boards can go to 1TB, but about 480GB was tested at the time , using FAT32:

https://www.facebook.com/groups/1086267554764016/permalink/3613783035345776

The Reverb listing says that pics show it running with 480GB, check it out.

Of course, that's probably overkill , but pretty cool for a 1998 hardware sampler ...

**Reply by baritonomarchetto** [reply]
A picture and a guy on facebook claiming the AKAI recognized 400+ Gb is not solid enought for me to say all that storing space is actually usable, I am sorry :)

**Reply by temporal-distortion** [reply]
I hate to dissapoint you baritonmarchetto as i am part of that thread as well (being an owner of the S5000k sampler) and i have successfully taken it all the way up to 512gb of storage (was going to try 2TB but ended up slapping that drive into my laptop instead before i did that. Because what do you need 2TB of sample data for??).

The trick is to use this utility to override the limitations of FAT32 that windows will impose on you:

https://www.majorgeeks.com/files/details/fat32format.html#google_vignette
